forge-ai/src/main/java/forge/ai/simulation/GameStateEvaluator.java

```diff
@@ -148,7 +161,21 @@ public int evalCard(Game game, Player aiPlayer, Card c) {
         if (c.isCreature()) {
             return eval.evaluateCreature(c);
         } else if (c.isLand()) {
-            return 100;
+            int value = 100;
+            // for each mana color a land generates for free, increase the value by one
+            // for each mana a land can produce, add one hundred.
+            int max_produced = 0;
+            Set<String> colors_produced = new HashSet<>();
+            for (SpellAbility m: c.getManaAbilities()) {
+                m.setActivatingPlayer(c.getController());
+                max_produced = max(max_produced, m.amountOfManaGenerated(true));
+                for (AbilityManaPart mp : m.getAllManaParts()) {
+                    colors_produced.addAll(Arrays.asList(mp.mana(m).split(" ")));
+                }
+            }
+            value = 100 * max_produced;
+            value += colors_produced.size();
+            return value;
```

Comment:
the way you coded this means it will now never play something like _Maze of Ith_

Reply:
Added another term for activated abilities that are not mana abilities.